config/file_resolver.go

```diff
@@ -42,8 +42,11 @@ func NewRelativeResolver(paths ...string) FileResolver {
 
 	copy(pathList, paths)
 
-	// add the current cwd
-	if cwd, err := os.Getwd(); err == nil {
+	if appRoot := os.Getenv(EnvironmentPrefix() + _root); appRoot != "" {
```

Comment:
yeah, i wasn't sure what i wanted. i can hardcode it to `APP_ROOT` if we think that's best.